pxtrunner/runner.ts

```diff
@@ -109,7 +108,24 @@ namespace pxt.runner {
                         return Promise.resolve()
                     }
                     if (proto == "empty") {
-                        epkg.setFiles(emptyPrjFiles())
+                        if (Object.keys(epkg.files).length == 0) {
+                            epkg.setFiles(emptyPrjFiles())
+                        }
+                        if (deps) {
+                            let files = getEditorPkg(pkg).files;
+                                let cfg = JSON.parse(files[pxt.CONFIG_NAME]) as pxt.PackageConfig;
+                                deps.forEach((dep: string) => {
+                                    if (dep.indexOf("=") > 0) {
```

Comment:
seems like we do this in various other places, maybe have a helper to parse and populate the map?